NwPluginAPI/Core/Server.cs

```diff
@@ -18,24 +18,22 @@ namespace PluginAPI.Core
 	/// </summary>
 	public class Server : Player
 	{
-		private static bool _isInitialized;
+		private static Server _instance;
 
 		/// <summary>
-		/// Initializes server instance.
+		/// The <see cref="Server"/> instance.
 		/// </summary>
-		public static void Init()
+		public static Server Instance
 		{
-			if (_isInitialized) return;
```

Comment:
I'm pretty sure that only this line is the issue and it will be enough to remove this check and `_isInitialized` bool.

It shouldn't be needed to rewrite entire Instance variable.

Reply:
I couldn't not find where that Init() function is called so i opted for this solution
I will change that now

Reply:
`Init()` is called from the base game.

Changing it in your way has a slightly worse performance and requires changing the base game, which would delay the merge and require more extensive QA tests.

I'm pretty sure removing that if will be enough. If it's not enough then we will have to do it your way.

Reply:
I see..
Thanks for the explanation